satpy/etc/readers/fci_l2_nc.yaml

```diff
@@ -340,6 +401,14 @@ datasets:
     layer: 0
     standard_name: air_pressure_at_cloud_top
```

Comment:
Here I would add the `long_name` from the FCIL2FS but with the addition `"(upper layer)"` at the end

Reply:
appended "for upper/lower layer" to the long names